services/web/server/src/simcore_service_webserver/users/_preferences_models.py

```diff
@@ -105,9 +112,19 @@ class JobConcurrencyLimitFrontendUserPreference(FrontendUserPreference):
     UserInactivityThresholdFrontendUserPreference,
     JobConcurrencyLimitFrontendUserPreference,
     AllowMetricsCollectionFrontendUserPreference,
+    TelemetryLowDiskSpaceWarningThresholdFrontendUserPreference,
 ]
 
 
+def get_preference_identifier(
+    preference_class: type[FrontendUserPreference],
+) -> PreferenceIdentifier:
+    preference_identifier: PreferenceIdentifier = preference_class.__fields__[
+        "preference_identifier"
+    ].default
+    return preference_identifier
```

Comment:
Q: you are always returning a constant here, right? i.e. the default in `preference_identifier: Field(default=THIS-VALUE_HERE)` on the `preference_class`.

What is the database value for? I am not sure I understand the preference setup ... perhaps we can discuss offline?

Reply:
I see that this is a bit confusing, but this is not used directly in the db. In fact the `FrontendUserPreference` class has a `to_db` method which excludes this field.

The field is used by the fronted and it's a mapping between the "fronted identifier", in this case `preference_identifier` and the `preference_name` (the name of the class) used in the db.

I have also moved that code to the class definiton, since it makes senso to have it there.